fendermint/vm/genesis/Cargo.toml

```diff
@@ -26,7 +26,7 @@ fendermint_actor_eam = { workspace = true }
 
 fendermint_crypto = { path = "../../crypto" }
 fendermint_testing = { path = "../../testing", optional = true }
-fendermint_vm_core = { path = "../core" }
+fendermint_vm_core = { workspace = true }
```

Comment:
Why this change? What do we gain?

Reply:
oh, it's just using the workspace build, so that there is just one build, so the crates all use the same version. Should not have any effect.